File: PSET1/mario.js
//Pset1 - Mario

function pyramid(height) {
    let spaces = height - 1;
    let bricks = 2;
    const gap = ' ';
    const brick = '#';

    for (i = 0; i < height; i++) {
        let row = gap.repeat(spaces).concat(brick.repeat(bricks));
        console.log(row);
        spaces--;
        bricks++;
    }
    };

 function init(){
    var input = prompt('Please enter your pyramid height');
    //parseInt(input);
    if(input >= 0 && input < 24) {
        pyramid(input);
    } else {
        init();
    }
}

init();

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int input = get_int("Please enter your pyramid height: ");

    if (input >= 0 && input < 24) {

    for (int l = 0; l < input; l++){
        //To print the spaces(s)
        for(int s = input - l; s > 1; s--){
            printf(" ");
        }
        //To print the bricks(b)
        for (int b = 0; b< l+2; b++){
            printf("#");
        }
        //New line
            printf("\n");
        }

    } else {
        main();
    }

}